base/task.jl

```diff
@@ -345,9 +345,26 @@ end
 
 Wrap an expression in a [`Task`](@ref) and add it to the local machine's scheduler queue.
 
-Values can be interpolated into `@async` via `\$`, which copies the value directly into the
-constructed underlying closure. This allows you to insert the _value_ of a variable,
-isolating the aysnchronous code from changes to the variable's value in the current task.
+Values of a variable `x` can be interpolated into `@async` using `\$x`. This allows you 
+to insert the _value_ of a variable, isolating the aysnchronous code from changes to the 
+variable's value in the current task. However, note that interpolation does not 
+[`copy`](@ref) the variable value, i.e. if `x` is mutated instead of reassigned, these 
+changes will be visible in the `@async` block.
+
+# Examples
+```jldoctest
+julia> x = ["original"]
+       t = @async begin
+           sleep(0.1) # Wait for change in x
+           println(" x = ", x)
+           println("\$x = ", $x)
```

Comment:
Don't you need something like

```suggestion
           println("\\\$x = ", \$x)
```

here and other places?

Reply:
Well possible. How can I test these sort of things? I made all changes so far by simply editing the files on GitHub. Should I switch to a different workflow?

Reply:
I think it's OK to use a browser if you like, as long as the test suite is happy about it.  CONTRIBUTING.md has a section on how to check it locally <https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#modifying-an-existing-docstring-in-base>.

Reply:
Thanks for the link. I believe I fixed all the $ escaping now. One of the tests still comes back negative, but I'm not sure why.